Dockerfile.Windows

```diff
@@ -0,0 +1,31 @@
+FROM mcr.microsoft.com/dotnet/framework/runtime:4.8-windowsservercore-ltsc2019
```

Comment:
If an github action can be added, that would be even better; in that case, for future main branch changes, your Windows dockerfile setup will be automatically checked.